.github/workflows/inter-branch-merge-flow.yml

```diff
@@ -0,0 +1,13 @@
+name: Inter-branch merge workflow
+on:
+  push:
+    branches:
+      - release/**
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  Merge:
+    uses: dotnet/arcade/.github/workflows/inter-branch-merge-base.yml@main
```

Comment:
> @main

Does this mean any changes in the dotnet/arcade main branch will automatically get used here? Would it be better if there was some sort of "release" channel, so that only stabilized changes get pushed out to other repos?

Reply:
Thank you for the feedback! 
The communication about those PR's was not sent yet, it will soon. 

Yes, updates will be immediately available once merged, however this is something that is not expected to change a lot to be honest. 
The functionality of the merge logic was get from: https://github.com/dotnet/arcade/blob/main/scripts/GitHubMergeBranches.ps1 which didn't change for 3 years. 
However if users would like to freeze the version used, there is always a possibility to use the workflow from specified commit. 
`dotnet/arcade/.github/workflows/inter-branch-merge-base.yml@commit-ref`
The testing of the workflow happens before the merge into the main and done on private repositories.